src/app_service/service/wallet_account/service.nim

```diff
@@ -180,7 +180,7 @@ QtObject:
         account.relatedAccounts = accounts.filter(x => not account.derivedFrom.isEmptyOrWhitespace and (cmpIgnoreCase(x.derivedFrom, account.derivedFrom) == 0))
         self.walletAccounts[account.address] = account
 
-      self.buildAllTokens()
+      self.buildAllTokens(toSeq(self.walletAccounts.keys()))
```

Comment:
Nit: `toSeq(self.walletAccounts.keys())` seems to be repeated in several places. Perhaps we could make it a function and give it a friendly name like `getWalletAddresses`

Reply:
updated